lenet/lenet.js

```diff
@@ -0,0 +1,132 @@
+
+'use strict';
+
+const nn = navigator.ml.getNeuralNetworkContext('v2');
+
+function sizeOfShape(shape) {
+  return shape.reduce((a, b) => {
+    return a * b;
+  });
+}
+
+export class Lenet {
```

Comment:
should this class be named as LeNet?

Reply:
sounds good. I'l fix that.

Reply:
Fixed. Please take a look at the new commit.